includes/Admin/Admin_AJAX.php

```diff
@@ -278,6 +223,69 @@ public function run_checks() {
 		);
 	}
 
+
+	/**
+	 * Get the AJAX_Runner runner.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return AJAX_Runner
+	 */
+	protected function get_runner() {
+		$runner = Plugin_Request_Utility::get_runner();
+
+		if ( is_null( $runner ) ) {
+			$runner = new AJAX_Runner();
```

Comment:
@felixarntz @jjgrainger By doing this, should we not also be calling the prepare method on this runner?